Mathlib/AlgebraicTopology/MooreComplex.lean

```diff
@@ -153,8 +162,8 @@ which maps each of these intersections of kernels to the next.
 def normalizedMooreComplex : SimplicialObject C ⥤ ChainComplex C ℕ where
   obj := obj
   map f := map f
-  map_id X := by ext (_ | _) <;> aesop_cat
-  map_comp f g := by ext (_ | _) <;> apply Subobject.eq_of_comp_arrow_eq <;> aesop_cat
+  map_id X := by ext (_ | _) <;> dsimp <;> aesop_cat
+  map_comp f g := by ext (_ | _) <;> apply Subobject.eq_of_comp_arrow_eq <;> dsimp <;> aesop_cat
```

Comment:
Could we have a note here? I'm confused why `aesop_cat` can't do the `dsimp` step.